cogs/reddit.py

```diff
@@ -0,0 +1,441 @@
+from discord import Embed, Color
+from discord.ext.commands import Cog, Bot, command, Context, is_nsfw
+import praw
+import os
+import random
+from .utils import constants
+
+reddit = praw.Reddit(
+    client_id=os.getenv("REDDIT_CLIENT_ID"),
+    client_secret=os.getenv("REDDIT_CLIENT_SECRET"),
+    user_agent=os.getenv("REDDIT_USER_AGENT"),
+    username=os.getenv("REDDIT_USERNAME")
+)
+
+
+class Reddit(Cog):
+    def __init__(self, bot: Bot) -> None:
+        self.bot = bot
+
+    @command()
+    async def memes(self, ctx: Context) -> None:
+        name = random.choice(constants.meme_sublist)
+        subreddit = reddit.subreddit(name)
+
+        hotposts = subreddit.hot(limit=100)
+        postlist = list(hotposts)
```

Comment:
There is no reason for `hotposts` variable to exist when it's only used to be converted into `postlist`, use the variable as a list right away instead.

**NOTE:** You'll need to change all references to `postlist` to `hotposts`
**APPLIES:** This applies to every occurence like this, lines: 55, 92, 129, 167, 197, 227, 264, 371
```suggestion
        hotposts = list(subreddit.hot(limit=100))
```